lib/ansible/cli/galaxy.py

```diff
@@ -624,6 +624,12 @@ def parse_role_req(requirement):
 
         else:
             # Newer format with a collections and/or roles key
+
+            if 'roles' in file_requirements and file_requirements['roles'] is None:
+                file_requirements['roles'] = []
+            if 'collections' in file_requirements and file_requirements['collections'] is None:
+                file_requirements['collections'] = []
```

Comment:
I'd probably add `or []` on line 647 instead to match the style used for roles below.

Reply:
Fixed that, thanks.